spot_driver/src/api/spot_image_sources.cpp

```diff
@@ -97,32 +105,43 @@ tl::expected<ImageSource, std::string> fromSpotImageSourceName(const std::string
   }
 }
 
+std::set<SpotCamera> getSpotCamerasUsed(const bool has_hand_camera, const std::vector<std::string>& cameras_used) {
```

Comment:
Definitely interested in this, but I am worried it might lead to a lot of weird camera specific dependencies in the parameter interface implementation (e.g. where to then put kRosStringToSpotCamera)

Reply:
I went ahead and implemented this change -- now createImageSources takes in a set of SpotCameras.